src/UICircularProgressRing/UICircularProgressRing.swift

```diff
@@ -1009,6 +1010,8 @@ fileprivate extension CALayer {
         delegate?.willDisplayLabel?(for: self, label)
     }
 
+    fileprivate var completionTimer: Timer?
```

Comment:
use `private` instead

Reply:
I changed it but what is your reasoning behind using private here instead of fileprivate? These variables should only be kept inside the UICircularProgressRing and never be accessible outside of it or by any subclasses I would assume (which is why I made them fileprivate)

Reply:
If that is the intent then `private` conveys that more clearly. By making them `fileprivate` if a new class or object was added into this file those members would be accessible from those new objects. In general you should prefer `private` over `fileprivate` for most cases. Take a look at the default `swiftlint` rules for more reasoning.